packages/components/src/styles/tag-components.scss

```
@@ -16,24 +23,32 @@
 
```

```suggestion
&[data-emphasis] {
	@include overwrite-strong-white();
}
```

Probably I'm misunderstanding something here, but wouldn't this declaration be simpler?

---

But we need this only for [data-emphasis="strong"]

And this color would be overwritten by the following [data-variant="critical"][data-emphasis="strong"]. So I had to included it everytime